CHANGELOG.md

```diff
@@ -19,52 +19,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-17
 
+Amazon Chime SDK React Components Library v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK React Components Library v3 includes major improvements for component style customization, `MeetingProvider/MeetingManager` configuration, device management, WebRTC metrics and logger.
+
+- **Component style customization**: Improve components for easier customization of style. Publish this [styling guide](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/styling-guide--page) to instruct on how to customize the component style.
+- **`MeetingProvider/MeetingManager` configuration**: Remove most of `Props` of `MeetingProvider` and improve `MeetingManager.join()` API to allow configuring `MeetingSession` and some other options (e.g., `enableWebAudio`, `logger`) before joining a meeting instead of when `MeetingProvider` is mounted.
+- **Device management**: Update the type of most `device` from `string | null` to `Device | TransformDevice` for better `TransformDevice` support. Remove redundant hooks `useSelectAudioInputDevice`, `useSelectVideoInputDevice`, and `useSelectAudioOutputDevice`. Rename device selection methods of `MeetingManager` and now they throw errors when failed.
+- **WebRTC metrics**: Publish the standardized WebRTC metrics for all supported browsers. Deprecate `useBandwidthMetrics` hook as duplicate of `useMediaStreamMetrics` hook.
```

Comment:
Deprecate `useBandwidthMetrics` hook as duplicate of `useMediaStreamMetrics` hook. to 
Deprecate `useBandwidthMetrics` hook in favor of `useMediaStreamMetrics` hook.

Reply:
Updated!